src/naive/datetime/mod.rs

```diff
@@ -127,6 +127,28 @@ impl NaiveDateTime {
         datetime.expect("invalid or out-of-range datetime")
     }
 
+    /// Makes a new `NaiveDateTime` corresponding to a UTC date and time,
+    /// from the number of milliseconds
+    /// since the midnight UTC on January 1, 1970 (aka "UNIX timestamp").
+    ///
+    /// Returns `None` on the out-of-range number of milliseconds and/or invalid nanosecond.
```

Comment:
In this case, there's no "invalid nanosecond", right? Also I think "the" -> "an" would read better?

Reply:
No, there are no invalid nanos in this case. Changed.